templates/config_server.json.j2

```diff
@@ -17,7 +17,7 @@
   {% endif -%}
   "bootstrap": false,
   {% if consul_bootstrap_expect -%}
-    "bootstrap_expect": {{ consul_servers|length }},
+    "bootstrap_expect": {{ _consul_lan_servers|length }},
```

Comment:
@brianshumate do you want to have spaces in these cases to?

`{% if _consul_wan_servers|length > 0 -%}`

Or only when the result is not used in an expression?